recipe/meta.yaml

```diff
@@ -108,6 +108,9 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
```

Comment:
You need to copy this to `faiss` output too

Reply:
Thanks a lot, I overlooked that!